airflow/www/views.py

```diff
@@ -798,11 +798,14 @@ def log(self):
         if PY2 and not isinstance(log, unicode):
             log = log.decode('utf-8')
 
+        if request.is_xhr:
```

Comment:
ok. this code is still a bit weird so I suggest adding comments pointing out
- `log.html` (the page this view renders) sometimes does an XHR request to itself, and we handle it here 
- link to http://flask.pocoo.org/docs/0.11/api/#flask.Request.is_xhr since this is a nonstandard feature

LGTM otherwise